packages/app/src/components/SearchBar/index.js

```diff
@@ -66,21 +67,53 @@ const SearchBar = () => {
 
   return (
     <>
-      <form onSubmit={handleSubmit}>
-        <fieldset>
-          <input
-            type="text"
-            name="search"
-            placeholder="  Search"
-            data-id="search-input"
-            onChange={handleInputChange}
-            aria-label="search-input"
-          />
-          <div data-id="error-message">
-            {showErrorMessage ? <>Application Not Found</> : null}
-          </div>
-        </fieldset>
-      </form>
+      <section className="searchBarWrapper">
```

Comment:
Nitpick: Typically, CSS properties follow `kebab-case` instead of `camelCase`, and I think it makes sense to follow that convention for all our `classNames`.

That said, if we're already using `camelCase` everywhere for classnames then feel free to ignore this feedback -- this is very minor and inconsequential.